lang/en.json

```
@@ -578,6 +578,8 @@
         "MatomoConfigurationMatomoDomainsException": "Value needs to be an array",
         "MatomoConfigurationMatomoAlwaysUseSendBeaconTitle": "Always use sendBeacon",
         "MatomoConfigurationMatomoAlwaysUseSendBeaconDescription": "Enables sendBeacon usage instead of a regular ajax request. This means when a user clicks for example on an outlink, the navigation to this page will happen much faster.",
+        "MatomoConfigurationMatomoDisableAlwaysUseSendBeaconTitle": "Disable always use sendBeacon permanently",
+        "MatomoConfigurationMatomoDisableAlwaysUseSendBeaconDescription": "Disable sendBeacon usage instead of a regular ajax request. This means when a user clicks for example on an outlink, the navigation to this page will be slower, but it will not be blocked by an Adblocker.",
```

@Orgoth Some text suggestion 
```suggestion
        "MatomoConfigurationMatomoDisableAlwaysUseSendBeaconDescription": "Choose to stop using sendBeacon and use a regular AJAX request instead. This might make clicking on external links a bit slower for users, but it won't be stopped by ad blockers.",
```